packages/cli/src/ActiveExecutions.ts

```diff
@@ -156,7 +157,11 @@ export class ActiveExecutions {
 
 		execution.workflowExecution!.cancel();
 
-		return await this.getPostExecutePromise(executionId);
+		// Reject all the waiting promises
+		const reason = new ExecutionCancelledError(executionId);
```

Comment:
Maybe out of scope, but currently we have
- `ExecutionCancelledError` as a rejection reason
- `new WorkflowOperationError('Workflow-Execution has been canceled!')` added to execution data at `ExecutionRepository.stopDuringRun`
- No error added to execution data at `ExecutionRepository.stopBeforeRun`
- `new WorkflowOperationError('Workflow-Execution has been canceled!')` in the `PCancelable` at `WorkflowRunner`

We should eventually make this consistent.